lib/node_modules/@stdlib/math/base/complex/exp/test/test.js

```diff
@@ -0,0 +1,274 @@
+
+	for ( i = 0; i < re.length; i++ ) {
+		q = cexp( re[ i ], im[ i ] );
+
+		if ( q[ 0 ] === expre[ i ] ) {
+			t.strictEqual( q[ 0 ], expre[ i ], 'returns expected real component' );
+		} else {
+			delta = abs( q[ 0 ] - expre[ i ] );
+			tol = EPS * abs( expre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. real: '+q[0]+'. expected: '+expre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( q[ 1 ] === expim[ i ] ) {
+			t.strictEqual( q[ 1 ], expim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = abs( q[ 1 ] - expim[ i ] );
+			tol = EPS * abs( expim[ i ] );
+			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. imag: '+q[1]+'. expected: '+expim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'the function computes exp(z) for complex z', function test( t ) {
+	var delta;
+	var expre;
+	var expim;
+	var tol;
+	var re;
+	var im;
+	var i;
+	var q;
+
+	re = generalComplex.re;
+	im = generalComplex.im;
+	expre = generalComplex.expre;
+	expim = generalComplex.expim;
+
+	for ( i = 0; i < re.length; i++ ) {
+		q = cexp( re[ i ], im[ i ] );
+
+		if ( q[ 0 ] === expre[ i ] ) {
+			t.strictEqual( q[ 0 ], expre[ i ], 'returns expected real component' );
+		} else {
+			delta = abs( q[ 0 ] - expre[ i ] );
+			tol = EPS * abs( expre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. real: '+q[0]+'. expected: '+expre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( q[ 1 ] === expim[ i ] ) {
+			t.strictEqual( q[ 1 ], expim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = abs( q[ 1 ] - expim[ i ] );
+			tol = EPS * abs( expim[ i ] );
+			t.ok( delta <= tol, 'within tolerance. z: '+re[i]+'+ '+im[i]+'i. imag: '+q[1]+'. expected: '+expim[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+	}
+	t.end();
+});
+
+tape( 'if imaginary component is `+Infinity`, the function returns `[NaN, NaN]`', function test( t ) {
+	var v = cexp( 0.0, PINF );
+	t.strictEqual( isnan( v[ 0 ] ), true, 'returns NaN' );
+	t.strictEqual( isnan( v[ 1 ] ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'if imaginary component is `-Infinity`, the function returns `[NaN, NaN]`', function test( t ) {
+	var v = cexp( 0.0, NINF );
+	t.strictEqual( isnan( v[ 0 ] ), true, 'returns NaN' );
+	t.strictEqual( isnan( v[ 1 ] ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'if real component is `-Infinity`, the function returns `[0.0, 0.0]`', function test( t ) {
+	var v = cexp( NINF, 1.0 );
+	t.strictEqual( v[ 0 ], 0.0, 'returns 0.0' );
+	t.strictEqual( v[ 1 ], 0.0, 'returns 0.0' );
+	t.end();
+});
+
+tape( 'if real component is `+Infinity` and imaginary component is `0.0`, the function returns `[+Infinity, 0.0]`', function test( t ) {
+	var v = cexp( PINF, 0.0 );
+
+	// The real component is computed as Infinity * 1.0 = Infinity:
+	t.strictEqual( v[ 0 ], PINF, 'returns +Infinity' );
+
+	// The imaginary component is computed as Infinity * 0.0 = NaN:
```

Comment:
@rreusser Slightly confused by this comment. The test asserts that the imaginary component is `0`, while the comment leads me to believe that the result should be `NaN`. Am I missing something?

Same thing for L175.

Reply:
This is the problem with comments that only reiterate what the code does: you fix the code and fail to update the comments. I've removed the comments in these two tests. If anything it'd be far better to link back to the source of these choices rather than than simply reiterate what the test itself says.